pkg/cluster/builder.go

```diff
@@ -104,16 +119,45 @@ func (b FileSpecBuilder) getConfig(clusterConfigURL string) (*Config, error) {
 	return ParseConfig(yaml)
 }
 
-func (b FileSpecBuilder) getBundles() (*releasev1.Bundles, error) {
+func (b FileSpecBuilder) createManifestReader() *manifests.Reader {
+	var opts []manifests.ReaderOpt
+	if b.releasesManifestURL != "" {
+		opts = append(opts, manifests.WithReleasesManifest(b.releasesManifestURL))
+	}
+	return manifests.NewReader(b.reader, opts...)
+}
+
+func (b FileSpecBuilder) getBundles(manifestReader *manifests.Reader) (*releasev1.Bundles, error) {
 	bundlesURL := b.bundlesManifestURL
 	if bundlesURL == "" {
-		var opts []manifests.ReaderOpt
-		if b.releasesManifestURL != "" {
-			opts = append(opts, manifests.WithReleasesManifest(b.releasesManifestURL))
-		}
-		manifestReader := manifests.NewReader(b.reader, opts...)
 		return manifestReader.ReadBundlesForVersion(b.cliVersion.GitVersion)
 	}
 
 	return bundles.Read(b.reader, bundlesURL)
 }
+
+func buildEKSARelease(release *releasev1.EksARelease, bundle *releasev1.Bundles) *releasev1.EKSARelease {
+	version := strings.Split(release.Version, "+")[0]
```

Comment:
is it possible to include the full semver in the name?

Reply:
yes, changed it